packages/rs-sdk/src/platform/block_info_from_metadata.rs

```diff
@@ -0,0 +1,24 @@
+use crate::Error;
+use dapi_grpc::platform::v0::ResponseMetadata;
+use dpp::block::block_info::BlockInfo;
+use dpp::block::epoch::MAX_EPOCH;
+use drive::error::proof::ProofError;
+
+pub fn block_info_from_metadata(response_metadata: &ResponseMetadata) -> Result<BlockInfo, Error> {
```

Comment:
1. Missing docs
2. Why not `TryFrom` trait?

Reply:
Docs done, we can't use try_from because both of these are in different crates.